indico/modules/events/abstracts/fields.py

```diff
@@ -209,7 +211,16 @@ def permissions_info(self):
 
     @property
     def event_roles(self):
-        return [serialize_role(role, legacy=False) for role in sorted(self.event.roles, key=attrgetter('code'))]
+        return [serialize_event_role(role, legacy=False) for role in sorted(self.event.roles, key=attrgetter('code'))]
+
+    @property
+    def category_roles(self):
+        category_roles = list(
```

Comment:
I found a much nicer way to get all available category roles for a category:

    CategoryRole.query.join(cat.chain_query.subquery()).all()

This would probably fit well in a `get_for_category(cat)` staticmethod of the `CategoryRole` model.